stop.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
 
+getresource() {
+  # 30% of cases are going to be targeted (if the list of resources is greater than 24)
+  if [[ $(wc -l resources.txt | awk "{print $1}") -gt 24 && $(shuf -i 1-100 -n 1) -le 30 ]]; then
```

Comment:
this line throws error in ubuntu, use `wc -l < resources.txt` instead `wc -l resources.txt`